modules/lfs/locks.go

```diff
@@ -13,19 +13,25 @@ import (
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/modules/setting"
 	api "code.gitea.io/sdk/gitea"
-
-	"gopkg.in/macaron.v1"
 )
 
-func checkRequest(req macaron.Request, post bool) int {
+func checkRequest(ctx *context.Context, post bool) int {
```

Comment:
Would it be cleaner to write responses to `ctx`, instead of returning the response status? Right now, the person calling this function needs to know that 200 is a special return value (which isn't documented anywhere)

Reply:
I have wrote the method.